.semaphore/semaphore.yml

```diff
@@ -2,89 +2,47 @@ version: v1.0
 name: workos-node pipeline
 
 execution_time_limit:
-  minutes: 15
+  minutes: 10
 
 agent:
   machine:
     type: e1-standard-2
     os_image: ubuntu1804
 
 global_job_config:
+  env_vars:
+    - name: NODE_ENV
+      value: test
   prologue:
     commands:
       - checkout
       - nvm use
+      - bash ~/workos-node/bin/restore-or-install
 
 blocks:
   - name: Install dependencies
+    dependencies: []
     task:
-      secrets:
-        - name: workos-npm
-      env_vars:
-        - name: NODE_ENV
-          value: test
-        - name: CI
```

Comment:
Unused?

Reply:
Yup, and we have `SEMAPHORE` available, which I think we should use instead.